inlong-dataproxy/dataproxy-docker/dataproxy-docker.sh

```diff
@@ -25,6 +25,8 @@ sed -i "s/manager.hosts=.*$/manager.hosts=${MANAGER_OPENAPI_IP}:${MANAGER_OPENAP
 sed -i "s/audit.proxys=.*$/audit.proxys=${AUDIT_PROXY_URL}/g" "${common_conf_file}"
 sed -i "s/proxy.report.ip=.*$/proxy.report.ip=${local_ip}/g" "${common_conf_file}"
 sed -i "s/proxy.cluster.tag=.*$/proxy.cluster.tag=${CLUSTER_TAG}/g" "${common_conf_file}"
+sed -i "s/proxy.cluster.name=.*$/proxy.cluster.name=${$CLUSTER_NAME}/g" "${common_conf_file}"
```

Comment:
Please add those configs in the `Dockerfile`.

Reply:
done